src/components/hero/HeroSection.css

```diff
@@ -27,18 +30,33 @@
     background: #FFD500;
     border: 1px solid #000000;
   }
+
   #hero-title {
-    margin-left: 24px;
-    width: 1245px;
-    height: 128px;
+    margin-top: 48px;
+    margin-left: 10%;
+    width: 84vw;
     font-family: sans-serif;
-    font-size: 56px;
+    font-size: 3.7vw;
     font-style: normal;
     font-weight: 800;
-    line-height: 64px;
+    line-height: 114.286%; 
     letter-spacing: 0.4px;
     color: var(--neutral-neutral-01, #18181A);
+    text-align: start;
   }
+    .p2 {
+      padding-bottom: 48px;
+      margin-left: 10%;
+      font-family: sans-serif;
+      font-size: 3.7vw;
+      font-style: normal;
+      font-weight: 800;
+      line-height: 114.286%;
```

Comment:
same question as above.

Reply:
font size need to be responsive when screen size change, but we still can discuss about it

line-height precision copied from Figma

Reply:
Same question as before, why not 4vw?